src/context/actions/dataActions.js

```diff
@@ -0,0 +1,42 @@
+export const videoConstants = {
+  LOADING: "VIDEO_LOADING",
+  GET_VIDEOS: "GET_VIDEOS",
+  GET_VIDEO: "GET_VIDEO",
+  ERROR: "VIDEO_ERROR",
+};
+
+export const likeConstants = {
+  LOADING: "LIKED_LOADING",
+  GET_LIKED: "GET_LIKED",
+  ERROR: "LIKE_ERROR",
+  ADD_TO_LIKED: "ADD_TO_LIKED",
+  REMOVE_FROM_LIKED: "REMOVE_FROM_LIKED",
+};
+
```

Comment:
Action variable separated for every reducer??

Reply:
I have one reducer, different action objects according to the action to be performed. Please check